ts/src/index.ts

```diff
@@ -116,25 +123,21 @@ const profiler: Profiler|undefined = undefined;
 
 /**
  * Initializes the config, and starts heap profiler if the heap profiler is
- * needed. Returns a profiler if creation is successful. Otherwise, returns
- * undefined.
+ * needed. Returns a profiler if creation is successful. Otherwise, throws an
+ * error.
```

Comment:
The return value is actually a promise, so this function never throws actually. It will always be a promise – a rejected promise in the case of a throw.

Reply:
changed comment.